src/Generator/ClassGenerator.php

```diff
@@ -446,10 +446,9 @@ public function removeExtentedClass()
      */
     public function setImplementedInterfaces(array $implementedInterfaces)
     {
-        array_map(function ($implementedInterface) {
-            // This loop is just validating that the given `$implementedInterfaces` contains valid syntax/symbols
-            return TypeGenerator::fromTypeString($implementedInterface);
-        }, $implementedInterfaces);
+        // This loop is just validating that the given `$implementedInterfaces` contains valid syntax/symbols
+        array_map(static fn($implementedInterface) =>
+            TypeGenerator::fromTypeString($implementedInterface), $implementedInterfaces);
```

Comment:
any way to suppresss this psalm error: `The call to array_map is not used` ? I tried with : 

```bash
@psalm-suppress UnusedFunctionCall
```

but it seems still error, even added to `psalm.xml`:

```xml
        <UnusedFunctionCall>
            <errorLevel type="error">
                <referencedFunction name="UnusedFunctionCall"/>
            </errorLevel>
        </UnusedFunctionCall>
```